README.md

```diff
@@ -1,27 +1,24 @@
-# Welcome to the Clarity project!
+# Clarity
+Clarity is a client-side JavaScript library that helps you understand how users view and use your webpage across all devices and browsers. 

```

Comment:
>all devices and browsers [](start = 113, length = 24)

Maybe 'all modern devices and browsers' would be better wording? There are certain required APIs, without which Clarity won't activate.